CIScripts/Testenv/Testenv.ps1

```diff
@@ -22,32 +22,49 @@ class ControllerConfig {
 }
 
 class SystemConfig {
-    [string] $AdapterName;
-    [string] $VHostName;
-    [string] $MgmtAdapterName;
-    [string] $ForwardingExtensionName;
+    [string] $AdapterName
+    [string] $VHostName
+    [string] $MgmtAdapterName
+    [string] $ForwardingExtensionName
 
     [string] VMSwitchName() {
         return "Layered " + $this.AdapterName
     }
 }
 
+class TestbedConfig {
+    [string] $Name
+    [string] $VMName
+    [string] $Address
+    [string] $Username
+    [string] $Password
+}
+
 class TestenvConfigs {
-    [SystemConfig] $System;
-    [OpenStackConfig] $OpenStack;
-    [ControllerConfig] $Controller;
+    [SystemConfig] $System
+    [OpenStackConfig] $OpenStack
+    [ControllerConfig] $Controller
+    [TestbedConfig[]] $Testbeds
+
+    TestenvConfigs([String] $TestenvConfFile) {
```

Comment:
I am not sure about this constructor. My thinking:
1) create one TestenvConfigs object in test file,
2) pass it's fields to functions that need config/

Other way would be just to remove this class and create in test 4 objects, one for each config type. I don't want the whole TestenvConfigs object to be passed around.